src/main/kotlin/net/irisshaders/lilybot/api/pluralkit/PluralKit.kt

```diff
@@ -46,6 +46,17 @@ object PluralKit {
 	 */
 	suspend fun checkIfProxied(id: Snowflake) = checkIfProxied(id.toString())
 
+	/**
+	 * Using a provided message ID, we call [getProxiedMessageAuthorId] to find out the author of the message.
+	 *
+	 * @param id The ID of the message being checked
+	 * @return The ID of the message author or null
+	 * @see getProxiedMessageAuthorId
+	 * @author NoComment1105
+	 * @since 3.3.2
+	 */
+	suspend fun getProxiedMessageAuthorId(id: Snowflake) = getProxiedMessageAuthorId(id.toString())
```

Comment:
Why do we have a whole function for a single .toString() call?

Reply:
Code cleanliness really, it's done for the other function too

Reply:
I guess so it looks cleaner where the method is used - don't need to add a toString there.

Reply:
I genuinely don't see what's clean about that. Take a snowflake, run it through a .toString(), pass it to the PK API. Having a whole additional method and kdoc alongside seems way messier.

Reply:
If you say so